packages/react-components/src/components/Survey/TagsSurvey/TagsSurvey.tsx

```diff
@@ -118,10 +125,10 @@ export const _TagsSurvey = (props: _TagsSurveyProps): JSX.Element => {
   }, [callIssuesToTag]);
 
   const onChange = React.useCallback(
-    (issueCategory: string, checked: boolean, issue?: string): void => {
+    (issueCategory: keyof SurveyIssues, checked: boolean, issue?: string): void => {
```

Comment:
use `_IssueCategory` here instead, then should be able to remove the explicit typing on setSelectedTags below